flash/pointcloud/detection/open3d_ml/app.py

```diff
@@ -148,8 +148,8 @@ def show_predictions(self, predictions):
 
                 viz = Visualizer()
                 lut = LabelLUT()
-                for id, color in dataset.color_map.items():
-                    lut.add_label(id, id, color=color)
+                for idx, color in dataset.color_map.items():
+                    lut.add_label(idx, idx, color=color)
```

Comment:
If: https://github.com/PyTorchLightning/lightning-flash/pull/1212#discussion_r820202531

```suggestion
                for id_color, color in dataset.color_map.items():
                    lut.add_label(id_color, id_color, color=color)
```